code/modules/surgery/organ_manipulation.dm

```diff
@@ -18,23 +18,24 @@
 	species = list(/mob/living/carbon/alien/humanoid)
 	steps = list(/datum/surgery_step/saw, /datum/surgery_step/incise, /datum/surgery_step/retract_skin, /datum/surgery_step/saw, /datum/surgery_step/manipulate_organs)
 
-
-
-
 /datum/surgery_step/manipulate_organs
 	time = 64
 	name = "manipulate organs"
 	implements = list(/obj/item/organ = 100, /obj/item/reagent_containers/food/snacks/organ = 0, /obj/item/organ_storage = 100, /obj/item/device/mmi = 100)
 	var/implements_extract = list(/obj/item/hemostat = 100, /obj/item/crowbar = 55)
 	var/implements_mend = list(/obj/item/cautery = 100, /obj/item/weldingtool = 70, /obj/item/lighter = 45, /obj/item/match = 20)
+	var/implements_heal = list(/obj/item/stack/medical/bruise_pack = 90, /obj/item/stack/medical/gauze = 55, /obj/item/stack/medical/gauze/improvised = 35)
+	var/implements_heal_robotic = list(/obj/item/screwdriver = 100)
```

Comment:
Nanopaste needs to be added to this list, but it doesn't exist yet.

Reply:
It exists now!
I kind of forgot to comment this a month ago.